ArduPlane/ArduPlane.cpp

```diff
@@ -45,7 +45,6 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(update_GPS_10Hz,        10,    400),
     SCHED_TASK(navigate,               10,    150),
     SCHED_TASK(update_compass,         10,    200),
-    SCHED_TASK(read_airspeed,          10,    100),
```

Comment:
calc_airspeed_errors() needed

Reply:
Added a task for calc_airspeed_error.

About the ordering of the tasks. From reading in the Scheduler, task order goes 
1. Plane tasks
2. AP_Vehicle tasks

So there are a bunch of tasks between calc_airspeed_error and & when Airspeed.read() gets called. I think the best I could in the current system is place  calc_airspeed_error  at the top of Plane tasks & Airspeed.read() at the bottom of AP_Vehicle tasks.

Advice here would be appreciated.